Contributing.md

```diff
@@ -82,4 +83,32 @@ to the TC and the TC uses its standard consensus seeking process to evaluate whe
 not to add this new member. Members who do not participate consistently at the level of
 a majority of the other members are expected to resign.
 
-
+# Project Captains
+
+The Express TC can designate captains for individual projects/repos in the
+organizations. These captains are responsible for being the primary
+day-to-day maintainers of the repo on a technical and community front.
+Repo captains are empowered with repo ownership and module publication rights.
+When there are conflicts, especially on topics that effect the Express project
+at large, captains are responsible to bring it up with the TC and drive
+those conflicts to resolution. Captains are also responsible in making sure
+community members follow the community guidelines, maintaining the repo
+and the registry, as well as in providing user support.
+
+To become a captain for a project you will be expected to participate in that
+project for at least 6 months prior to the request. You should have helped with
+code contributions as well as triaging issues. You are also required to have 2FA
+enabled on both your GitHub and npm accounts. When you want to request the
+captain role, submit a PR to this doc, under `Current Project Captains` section
+(maintaining the sort order) with the project and your GitHub handle.
+The PR will require at least 2 approvals from TC members and 2 weeks hold time to
+allow for dissent.  When the PR is merged, a TC member will add you to the proper
+GitHub groups, as well as grant you npm publish rights.
+
+## Current Project Captains
+
+- `expressjs.com`: @crandmck
+- `generator`: @wesleytodd 
+- `multer`: @LinusU 
+- `path-to-regexp`: @blakeembrey
+- `router`: @dougwilson & @wesleytodd
```

Comment:
This should be fixed to current `dougwilson`.

Reply:
done